src/components/SupportIndicator/variables.scss

```diff
@@ -0,0 +1,39 @@
+/* ----------------------------------------------
+* abstracts/_variables.scss
```

Comment:
Yes, these looks like identical variables like in CircularSupportIndicator, does not seem like good practice to have this file copy pasted to different components. @tobiastykvart I think it would be nice to resolve this variables thing in this PR, what do you think?